project/AppDependencies.scala

```diff
@@ -3,19 +3,19 @@ import sbt.*
 object AppDependencies {
 
   private lazy val bootstrapPlayVersion = "7.19.0"
-  private lazy val hmrcMongoVersion     = "1.3.0"
+  private lazy val hmrcMongoVersion     = "1.6.0"
 
   private val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc"                  %% "bootstrap-backend-play-28" % bootstrapPlayVersion,
     "com.github.java-json-tools"    % "json-schema-validator"     % "2.2.14",
     "uk.gov.hmrc.mongo"            %% "hmrc-mongo-play-28"        % hmrcMongoVersion,
-    "com.fasterxml.jackson.module" %% "jackson-module-scala"      % "2.15.2"
+    "com.fasterxml.jackson.module" %% "jackson-module-scala"      % "2.16.0"
   )
 
   private val test: Seq[ModuleID] = Seq(
-    "org.scalatest"         %% "scalatest"               % "3.2.16",
-    "com.typesafe.play"     %% "play-test"               % "2.8.20",
-    "org.mockito"           %% "mockito-scala-scalatest" % "1.17.14",
+    "org.scalatest"         %% "scalatest"               % "3.2.17",
+    "com.typesafe.play"     %% "play-test"               % "2.8.21",
```

Comment:
This should match the version of play you are using please update play in plugins.sbt.